dev/ColorPicker/ColorPicker.cpp

```diff
@@ -728,7 +737,7 @@ void ColorPicker::OnAlphaSliderValueChanged(winrt::IInspectable const& /*sender*
         return;
     }
 
-    UpdateColor(m_alphaSlider.Value() / 100.0, ColorUpdateReason::AlphaSliderChanged);
+    UpdateColor(m_alphaSlider.get().Value() / 100.0, ColorUpdateReason::AlphaSliderChanged);
```

Comment:
should there be an if check here first too?

Reply:
In general, my goal with this change is not to change any of the logic in the code, only to change the type of the reference.

In this case, an if check is not required, since the event handler should only fire if the alphaslider exists.

---
In reply to: [401110766](https://github.com/microsoft/microsoft-ui-xaml/pull/2191#discussion_r401110766) [](ancestors = 401110766)